execution_requests.go

```diff
@@ -0,0 +1,48 @@
+package compute
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+func (c *Client) Exec(runnable *Runnable, body io.Reader) ([]byte, *http.Response, error) {
+	req, err := c.execRequestBuilder(http.MethodPost, runnable.VersionPath(), body)
+
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "failed to Client.Exec")
+	}
+
+	res, err := c.do(req)
+	if err != nil {
+		return nil, res, errors.Wrap(err, "failed to Client.Exec")
+	}
+
+	result, err := ioutil.ReadAll(res.Body)
```

Comment:
`result` here is JSON containing two keys, we should unmarshal it into a struct

Reply:
I thought that was only if Atmo returns an error, otherwise it just returns the output bytes of the function directly.

Reply:
Yes sorry I got confused with the builder.Build output